eval_binop.go

```diff
@@ -80,10 +80,18 @@ func compareInterfaces(leftVal, rightVal any) (int, error) {
 		if rightString, ok := rightVal.(string); ok {
 			return strings.Compare(v, rightString), nil
 		}
+		if _, ok := rightVal.(*NullValue); ok {
```

Comment:
cant we put this check on line 78 and handle comparison for all the types vs null at once place?

Reply:
good point.